### DISCUSSION TITLE
What would be the label format?

### DISCUSSION THREAD

**Discussion by SeungyounShin** [ultralytics/ultralytics#564] [Q&A]
Training fails with multiples of image size and normalized label format with [cx,cy,w,h]

Do I have to resize the images before training?

**Comment by knoppmyth** [comment]
No, you shouldn't have to resize the images. You should post any error you're getting.

**Reply by glenn-jocher** [reply]
@knoppmyth for YOLOv8, the label format should be in the normalized `[class, cx, cy, w, h]` format, where `cx`, `cy`, `w`, and `h` are the center coordinates, width, and height of the bounding box, all normalized between 0 and 1 relative to the image size. The training process will automatically handle image resizing for you.

If you're encountering issues, please ensure your labels are correctly formatted and check your dataset's integrity. If the problem persists, feel free to share the error message you're receiving for further assistance. 🛠️
